src/components/SectionMain/SectionMain.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react';
+import { ToggleAll } from '../ToggleAll';
+import { TodoList } from '../TodoList';
+import { TodosContext } from '../TodosContext';
+
+export const SectionMain: React.FC = () => {
+  const { todos } = useContext(TodosContext);
+
+  if (todos.length) {
+    return (
+      <section className="main">
+        <ToggleAll />
```

Comment:
I think that logically ToggleAll should be a part of the panel (where's the input itself), not the main section, wdyt?

Reply:
I agree, which is logical, but I used html / css, which was provided in the task and did not change anything. If toggleAll move to another component, the appearance will break.